test/TestNullTasks.py

```diff
@@ -0,0 +1,8 @@
+def test_null_tasks(default_text_runner):
```

Comment:
Module docstring plz

Reply:
After https://github.com/ansible/ansible-lint/pull/900 it'll be linted automatically